src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h

```diff
@@ -47,11 +47,70 @@ namespace OpenMS
 {
 
   /**
-   * @brief Class to write out an OpenSwath OSW SQLite output (PyProphet input)
-   *
-   * The class can take a FeatureMap and create a set of string from it
-   * suitable for output to OSW using the prepareLine function.
-   *
+    @brief Class to write out an OpenSwath OSW SQLite output (PyProphet input).
+
+    The class can take a FeatureMap and create a set of string from it
+    suitable for output to OSW using the prepareLine function. The SQL data is
+    directly linked to the PQP file format described in the TransitionPQPFile class.
+    See also OpenSwathTSVWriter for another output format.
+
+    The file format has the following tables:
+
+      <table>
+        <tr> <th BGCOLOR="#EBEBEB" colspan=3>RUN</th> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">ID</td> <td>INT</td> <td> Primary Key (run id)</td> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">FILENAME</td> <td>TEXT</td> <td> Original filename associated with the run </td> </tr>
+      </table>
+
+      <table>
+        <tr> <th BGCOLOR="#EBEBEB" colspan=3>FEATURE</th> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">ID</td> <td>INT</td> <td> Primary Key (feature id)</td> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">RUN_ID</td> <td>INT</td> <td> Foreign Key (RUN.ID)</td> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">PRECURSOR_ID</td> <td>INT</td> <td> Foreign Key (TransitionPQPFile PRECURSOR.ID) </td> </tr>
```

Comment:
@grosenberger is this correct, the idea of this is that the PRECURSOR_ID relates to the TransitionPQPFile PRECURSOR.ID column, correct?

Reply:
Yes, that is correct.